addons/account/models/partner.py

```diff
@@ -538,6 +538,11 @@ def _get_company_currency(self):
     duplicated_bank_account_partners_count = fields.Integer(
         compute='_compute_duplicated_bank_account_partners_count',
     )
+    is_accounting_fields_required = fields.Boolean(compute="_compute_is_accounting_fields_required")
+
+    @api.depends('company_id')
+    def _compute_is_accounting_fields_required(self):
+        self.is_accounting_fields_required = bool(self.env.company.chart_template_id)
```

Comment:
```suggestion
        for partner in self:
            partner.is_accounting_fields_required = bool(partner.env.company.chart_template_id)
```
We need to iterate on the recordset :wink: